container-managed/pom.xml

```diff
@@ -42,14 +42,22 @@
     <dependency>
       <groupId>org.jboss.arquillian.junit</groupId>
       <artifactId>arquillian-junit-container</artifactId>
-      <!-- <scope>test</scope>   -->
```

Comment:
Notice it I just removed dead code :) It's been like that. But it's a good question in general. I will investigate